pyqt5ac.py

```diff
@@ -167,9 +174,10 @@ def main(rccOptions='', uicOptions='', force=False, config='', ioPaths=()):
             filename, ext = os.path.splitext(basename)
 
             # Replace instances of the variables with the actual values from the source filename
-            destFilename = destFileExpr.replace('%%FILENAME%%', filename) \
-                .replace('%%EXT%%', ext[1:]) \
-                .replace('%%DIRNAME%%', dirname)
+            destFilename = destFileExpr
+            variables.update({'FILENAME': filename, 'EXT': ext[1:],'DIRNAME': dirname})
```

Comment:
now if someone declared these variables, they would be replaced and therefore omitted silently.
I think we can consider these 3 - filename, ext, dirname - as predefined special variables which cannot be overridden by the user.

So maybe let's raise an error if any of these 3 are declared as variables?